docs/js/custom.js

```
@@ -0,0 +1,140 @@
+// © Copyright year. Last updated on Month date, year.
+document.addEventListener("DOMContentLoaded", function () {
+    let today = new Date();
+    let year = today.getFullYear();
+    let formattedDate = today.toLocaleDateString('en-US', {
+        year: 'numeric', month: 'short', day: 'numeric'
+    });
+
+    let copyrightText = `<p>© Copyright ${year}. Last updated on ${formattedDate}.</p>`;
+
+    // Select the <div role="contentinfo"> where the copyright should appear before
+    let contentInfo = document.querySelector("footer div[role='contentinfo']");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'contentInfo' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)